src/pylorax/installer.py

```diff
@@ -389,7 +389,7 @@ def novirt_install(opts, disk_img, disk_size, cancel_func=None):
             log.info(line)
 
         # Make sure the new filesystem is correctly labeled
-        setfiles_args = ["-e", "/proc", "-e", "/sys", "-e", "/dev",
+        setfiles_args = ["-e", "/proc", "-e", "/sys",
```

Comment:
Sorry I didn't mention this earlier, but `/proc` and `/sys`  appear to be borked too:
```
lrwxrwxrwx.   1 root root system_u:object_r:bin_t:s0            7 Jul 13  2018 bin -> usr/bin
dr-xr-xr-x.   6 root root system_u:object_r:boot_t:s0        4096 Jan 14 03:31 boot
drwxr-xr-x.   2 root root system_u:object_r:unlabeled_t:s0   4096 Jan 14 03:24 dev
drwxr-xr-x. 140 root root system_u:object_r:etc_t:s0        12288 Jan 15 05:37 etc
drwxr-xr-x.   3 root root system_u:object_r:home_root_t:s0   4096 Jan 15 08:38 home
lrwxrwxrwx.   1 root root system_u:object_r:lib_t:s0            7 Jul 13  2018 lib -> usr/lib
lrwxrwxrwx.   1 root root system_u:object_r:lib_t:s0            9 Jul 13  2018 lib64 -> usr/lib64
drwx------.   2 root root system_u:object_r:lost_found_t:s0 16384 Jan 14 03:23 lost+found
drwxr-xr-x.   2 root root system_u:object_r:mnt_t:s0         4096 Jul 13  2018 media
drwxr-xr-x.   2 root root system_u:object_r:mnt_t:s0         4096 Jul 13  2018 mnt
drwxr-xr-x.   2 root root system_u:object_r:usr_t:s0         4096 Jul 13  2018 opt
drwxr-xr-x.   2 root root system_u:object_r:unlabeled_t:s0   4096 Jan 14 03:24 proc
dr-xr-x---.   4 root root system_u:object_r:admin_home_t:s0  4096 Jan 15 10:06 root
drwxr-xr-x.   2 root root system_u:object_r:var_run_t:s0     4096 Jan 14 03:24 run
lrwxrwxrwx.   1 root root system_u:object_r:bin_t:s0            8 Jul 13  2018 sbin -> usr/sbin
drwxr-xr-x.   2 root root system_u:object_r:var_t:s0         4096 Jul 13  2018 srv
drwxr-xr-x.   2 root root system_u:object_r:unlabeled_t:s0   4096 Jan 14 03:24 sys
drwxrwxrwt.   5 root root system_u:object_r:tmp_t:s0         4096 Jan 14 03:31 tmp
drwxr-xr-x.  12 root root system_u:object_r:usr_t:s0         4096 Jan 14 03:26 usr
drwxr-xr-x.  22 root root system_u:object_r:var_t:s0         4096 Jan 14 03:31 var
```
Everything else looks good. I think all `-e`s should be dropped.